vochain/scrutinizer/db.go

```diff
@@ -0,0 +1,48 @@
+package scrutinizer
+
+import (
+	"time"
+
+	"github.com/timshannon/badgerhold/v3"
+	"go.vocdoni.io/proto/build/go/models"
+)
+
+type Process struct {
```

Comment:
Is it possible to avoid this definitions and use the protobuf models created? Badgerhold uses `encoding/gov` as a default encoding mechanism but it can be changed for your desired one on badgerhold.Options.

Reply:
What do you mean? Envelope, Mode and VoteOpts already use the protobuf types. For the rest I don't see the need since adding the Proto types means adding more complexity to the database and also size. I.e Status is a int32 type in the protobuf models but with some extra fields which are only required for Protobuf (some of them are also not exported). If you need to have the protobuf type, a simple conversion type is enough (models.Status(int32)).

Reply:
Ah, now I get what you mean. Using Protobuf as serialization mechanism for the whole badgerHold database.

TBH I'm not sure that's a good idea. That means we need to define the static types on a Protobuf types thus we loose flexibility for performing changes and we are also very limited to the Protobuf v3 types (which are not as rich as all GoLang types). 

If Protobuf encoding is much better than encoding/gov for the kind of types we use in therms of performance and size, then it might be worth it. However I'd let this for a future refactor since it requires some benchmarking and research. So we can add a new issue in order to do it in the future but I don't think its a good idea to do it on this very same PR (which is already quite big and risky).

Reply:
Yes I am referring to use protobuf for the whole db.
I am fine to tackle this on another moment and we can do a more in depth analysis.

Thank you for the explanation !